scripts/assets/file_content_validation_checks.textproto

```diff
@@ -36,6 +36,13 @@ file_content_checks {
   exempted_file_name: "domain/src/main/java/org/oppia/android/domain/oppialogger/logscheduler/MetricLogSchedulingWorker.kt"
   exempted_file_name: "scripts/src/javatests/org/oppia/android/scripts/regex/RegexPatternValidationCheckTest.kt"
 }
+file_content_checks {
+  file_path_regex: ".+?\\.xml"
+  prohibited_content_regex: "<include"
+  failure_message: "Remove <include .../> tag from layout and use the appropriate widget, e.g AppBarLayout widget instead."
```

Comment:
```suggestion
  failure_message: "Remove <include .../> tag from layouts and instead use the widget directly, e.g. AppBarLayout."
```

This adjustment might be a bit clearer.

Reply:
The suggested adjustment has been implemented with the latest change. PTAL